package.json

```diff
@@ -9,7 +9,8 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "cypress:open": "cypress open",
```

Comment:
Maybe it's just me, but I wonder if there are different names we can use that could convey the intent for each option better to someone not very familiar with Cypress. I can't remember what the difference is between `open` and `run` and we *just* talked about it. Maybe if one command said "ci" or "headless" or something? Or you might have an even better idea for phrasing.

Reply:
Good point! I like `ci` and I thought perhaps `open` could change to `local`